bin/apisix

```diff
@@ -103,8 +103,10 @@ worker_shutdown_timeout 3;
 {% if stream_proxy then %}
 stream {
     lua_package_path  "$prefix/deps/share/lua/5.1/?.lua;/usr/share/lua/5.1/apisix/lua/?.lua;]=]
+                      .. [=[/usr/local/apisix/deps/share/lua/5.1/apisix/lua/?.lua;]=]
```

Comment:
How about `[=[$prefix/deps/share/lua/5.1/apisix/lua/?.lua;]=]`?

I think this is better.